app/components/input.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+function Input() {
```

Comment:
Question: Same as button.  Thoughts around reusing a component from the library?